.github/tests/wpcm.spec.ts

```diff
@@ -11,7 +11,11 @@ test("homepage loads and contains example content", async ({ page }) => {
 });
 
 test("WP REST API is accessible", async ({ request }) => {
-  const apiRoot = await request.get(`${siteUrl}/wp-json`);
+  let apiRoot = await request.get(`${siteUrl}/wp-json`);
+  // If the api endpoint isn't /wp-json, it should be /wp/wp-json. This will probably be the case for main sites on subdirectory multisites.
+  if (!apiRoot.ok()) {
+    apiRoot = await request.get(`${siteUrl}/wp/wp-json`);
+  }
   expect(apiRoot.ok()).toBeTruthy();
 });
```

Comment:
The only time this comes into play is in subdirectory multisite on the main site. The paths for graphql and wp-api are `/wp/graphql` and `/wp/wp-json` in either of those cases. I fought to try to drop the `/wp` but eventually gave up. It might be a core multisite thing actually. 

This check and the switching of the URLs is for when we're testing on the main site of a subdirectory multisite vs testing any of the subsites or subdomain multisite. We could go a step further and confirm that somehow (but I'm not sure how to do that from the outside with javascript).